### PR TITLE
warning for missing relation

### PR DESCRIPTION
**Pull request by viennaa**
When the vrops looses connection to vcenter the inventory still will receive return 200 codes assuming everything is correct. However, the child resources below the vcenter are staying empty causing metrics to be missing. This warning should indicate to look into this direction.
TODO: there needs to be an alert to cover this.

### CODE REVIEW

**Review by richardtief — APPROVED** [review]
Looks good. Just one small suggestion.

File: tools/Vrops.py [hunk 1]
```diff
@@ -157,6 +157,9 @@ def get_resources(self, target: str,
         if response.status_code == 200:
             try:
                 relations = response.json()["resourcesRelations"]
+                if not relations:
+                    resourcekinds_beautyfied = ', '.join(resourcekinds)
+                    logger.warning(f'No child relation returned for {resourcekinds_beautyfied} from adapter {adapterkind} for {target}.')
```
[the remark this answers is not in the document]
Comment (richardtief) [reply tools/Vrops.py:162]: We are also free to use our own code for alerting purposes. `9xx` or similar.

**Comment by richardtief** [comment]
@viennaa We should take that up as well.

File: tools/Vrops.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (viennaa) [reply tools/Vrops.py:162]: incorporated with different return code

**Pull request merged**